autopilot.go

```diff
@@ -29,8 +29,21 @@ func venerableAppName(appName string) string {
 	return fmt.Sprintf("%s-venerable", appName)
 }
 
-func getActionsForExistingApp(appRepo *ApplicationRepo, appName, manifestPath, appPath string) []rewind.Action {
+func getActionsForExistingApp(appRepo *ApplicationRepo, appName, manifestPath, appPath string, options AutopilotOptions) []rewind.Action {
 	return []rewind.Action{
+		// delete old version if it still exists
+		{
+			Forward: func() error {
+				appExists, err := appRepo.DoesAppExist(venerableAppName(appName))
+				fatalIf(err)
+				if(appExists) {
+					fmt.Println("Found old version of app running, deleting.")
```

Comment:
Looking at this again, may want another check around this. Perhaps the "keep-existing-app" flag needs to be set in order to allow us to delete the old version?

Currently the plugin fails immediately if a -venerable already exists - we may still want that behavior for the default usage.